packages/blockchain-wallet-v4-frontend/src/modals/Interest/AccountSummary/template.success.tsx

```diff
@@ -62,6 +63,16 @@ const AccountSummary: React.FC<Props> = (props) => {
   const interestBalanceStandard = convertBaseToStandard(coin, interestBalanceBase)
   const pendingInterestStandard = convertBaseToStandard(coin, pendingInterestBase)
 
+  const getInterestEligibility = () => {
+    const interestEligibility = interestEligible[coin]
+    if (!interestEligibility) {
+      return false
+    }
+    return interestEligibility.eligible
+  }
+
+  const isDepositEnabled = getInterestEligibility()
```

Comment:
this isnt very performant. defining a function on every render and then invoking it. why not just the following, i believe it is the same result

`const isDepositEnabled = interestEligible[coin] ? interestEligible[coin]?.eligible : false`

Reply:
Sure, that also does it, thanks andrew. I was worried about the different structure of the response when it's eligible and when it's not but that does the trick. Will change it.